src/app/store/accounts.slice.test.ts

```diff
@@ -104,3 +117,24 @@ describe('removeAccountWorker()', () => {
       .silentRun();
   });
 });
+
+describe('generateAccountWorker', () => {
+  it('generates an account', () => {
+    return expectSaga(generateAccountWorker)
+      .provide([[call.fn(ipcBridgeRenderer.crypto.invoke), 'foo bar']])
+      .call(ipcBridgeRenderer.crypto.invoke, {
+        type: CryptoRequestType.CREATE_WALLET,
+        wallet: WalletType.MNEMONIC
+      })
+      .call(ipcBridgeRenderer.crypto.invoke, {
+        type: CryptoRequestType.GET_ADDRESS,
+        wallet: {
+          walletType: WalletType.MNEMONIC,
+          path: DEFAULT_DERIVATION_PATH,
+          mnemonicPhrase: 'foo bar'
+        }
+      })
+      .put(setGeneratedAccount({ mnemonicPhrase: 'foo bar', address: 'foo bar' as TAddress }))
```

Comment:
Can we do two different values here just to be certain? 😆

Reply:
I couldn't find how to `provide` two different values for the same function call using `expectSaga`.